app/models/framework/definition/RM1070.rb

```diff
@@ -14,30 +14,30 @@ class Invoice < EntryData
         field 'Customer Organisation', :string, exports_to: 'CustomerName', presence: true
         field 'Customer URN', :integer, exports_to: 'CustomerURN', urn: true
         field 'Customer Invoice Date', :string, exports_to: 'InvoiceDate', ingested_date: true
-        field 'Invoice Number', :string, exports_to: 'InvoiceNumber'
+        field 'Invoice Number', :string, exports_to: 'InvoiceNumber', presence: true
         field 'Invoice Line Number', :string
-        field 'Vehicle Model', :string, exports_to: 'ProductSubClass'
-        field 'Vehicle Make', :string, exports_to: 'ProductClass'
-        field 'Vehicle Segment', :string, exports_to: 'ProductGroup'
-        field 'UNSPSC', :string, exports_to: 'UNSPSC', ingested_numericality: { only_integer: true }, allow_nil: true
-        field 'Vehicle Registration Number', :string, exports_to: 'Additional1'
+        field 'Vehicle Model', :string, exports_to: 'ProductSubClass', presence: true
+        field 'Vehicle Make', :string, exports_to: 'ProductClass', presence: true
+        field 'Vehicle Segment', :string, exports_to: 'ProductGroup', presence: true
+        field 'UNSPSC', :string, exports_to: 'UNSPSC', ingested_numericality: { only_integer: true }
+        field 'Vehicle Registration Number', :string, exports_to: 'Additional1', presence: true
         field 'Unit of Purchase', :string, exports_to: 'UnitType'
-        field 'Invoice Price Per Vehicle', :string, exports_to: 'UnitPrice', ingested_numericality: true, allow_nil: true
-        field 'Quantity', :string, exports_to: 'UnitQuantity', ingested_numericality: { only_integer: true }, allow_nil: true
+        field 'Invoice Price Per Vehicle', :string, exports_to: 'UnitPrice', ingested_numericality: true
+        field 'Quantity', :string, exports_to: 'UnitQuantity', ingested_numericality: { only_integer: true }
         field 'Total Supplier price including standard factory fit options but excluding conversion costs and work ex VAT', :string, exports_to: 'InvoiceValue', ingested_numericality: true
-        field 'Additional Expenditure to provide goods', :string, exports_to: 'Expenses', ingested_numericality: true, allow_nil: true
-        field 'All Conversion and third party conversion costs excluding factory fit options', :string, exports_to: 'Additional2', ingested_numericality: true, allow_nil: true
+        field 'Additional Expenditure to provide goods', :string, exports_to: 'Expenses', ingested_numericality: true
+        field 'All Conversion and third party conversion costs excluding factory fit options', :string, exports_to: 'Additional2'
```

Comment:
Was the removal of `ingested_numericality` validation intentional?

(The template still has the shouty "THIS FIELD MUST ONLY CONTAIN NUMBERS TO 2 DECIMAL PLACES").

Reply:
oooh, this looks like a mistake. I'll re-run the checks and update.

Reply:
I've put this back to how it used to be.